### PR TITLE
add 'foldx3b6' as possible FoldX binary name

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyblocks/f/foldx.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def install_step(self):
             'foldx_%s.linux' % self.version,  # FoldX v2.x
             'FoldX.linux64',  # FoldX 3.x
             'foldx64Linux',  # FoldX 3.0-beta6
+            'foldx3b6',  # FoldX 3.0 beta 6.1 >_<
```
Comment (boegel) [inline easybuild/easyblocks/f/foldx.py]: deriving this from the version (`3.0-beta6.1`) is probably not worth the trouble...

**Comment by wpoely86** [comment]
nice one

**Comment by boegel** [comment]
@wpoely86: https://github.com/hpcugent/easybuild-easyconfigs/pull/1916 makes it even better...

**Comment by boegel** [comment]
Thanks @wpoely86!

**Pull request merged**